fdbserver/workloads/BlobGranuleVerifier.actor.cpp

```diff
@@ -342,10 +386,18 @@ struct BlobGranuleVerifierWorkload : TestWorkload {
 					// advance iterator before doing read, so if it gets error we don't retry it
 
 					try {
+						// before reading, prune at some version [0, readVersion)
```

Comment:
Simulation testing is probably going to be on hold until we have a solution for granule boundaries

Reply:
This is now a TODO